.travis.yml

```diff
@@ -36,6 +36,7 @@ before_install:
 
 install:
   - conda install --yes pip
+  - conda install --yes -c conda-forge geopandas
```

Comment:
installing `geopandas` for travis testing is unnecessary if `spaghetti` does not rely on `geopandas` as a dependency.

Reply:
I agree... That test wouldn't run if you did not install geopandas.

Reply:
Sounds great! Sorry that I did not realize `spaghetti` is relying on `geopandas`.

Reply:
@weikang9009 no problem at all. Please always give me feedback if you think something is amiss.